packages/protocol/README.md

```diff
@@ -57,3 +57,12 @@ The script will output two JSON files under `./deployments`:
 
 - `l2_genesis_alloc.json`: the `alloc` field which will be used in L2 genesis JSON file
 - `l2_genesis_storage_layout.json`: the storage layout of those pre-deployed contracts
+
+## Using Foundry
+
+This project also integrates with Foundry for building and testing contracts.
+
+- To compile using foundry: `forge build` or `pnpm compile2`
+- To run foundry tests: `forge test` or `pnpm test2`
```

Comment:
nit: I would recommend we name this pnpm script as compile:foundry and test:foundry, so the docs will says "pnpm test:foundry" and "pnpm compile:foundry"